src/cow-react/modules/swap/pure/EthFlow/EthFlowBanner/index.tsx

```diff
@@ -129,13 +128,20 @@ export function EthFlowBannerContent(props: EthFlowBannerContentProps) {
                     or{' '}
                     <u>
                       {' '}
-                      <SpanCta onClick={forceWrapCallback}>wrap my {native.symbol} and swap</SpanCta>
+                      <SpanCta
+                        onClick={(e) => {
+                          e.stopPropagation()
+                          wrapCallback(true)
+                        }}
+                      >
+                        wrap my {native.symbol} and swap
+                      </SpanCta>
                     </u>
                   </small>
                 </SpanCta>
               </>
             ) : (
-              <SpanCta onClick={wrapCallback}>Wrap my {native.symbol} and swap</SpanCta>
+              <SpanCta onClick={() => wrapCallback()}>Wrap my {native.symbol} and swap</SpanCta>
```

Comment:
Is the extra fn wrap needed?

```suggestion
              <SpanCta onClick={wrapCallback}>Wrap my {native.symbol} and swap</SpanCta>
```